Tests/Data/Parabolic/LiquidFlow/Dupuit/BC_BC/TestSet_01.prj

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
 <OpenGeoSysProject>
-    <meshes>
-        <mesh>square_1x1_quad_1e2.vtu</mesh>
-        <mesh>square_1x1_quad_1e2_geometry_left.vtu</mesh>
-        <mesh>square_1x1_quad_1e2_geometry_right.vtu</mesh>
-    </meshes>
```

Comment:
The indentation isn't correct. With the tool xmlstarlet it is possible to correct all format issues in the project file.

Reply:
Thanks, I will check all project files. However, the original files inside the Dupuit Folder should have been be deleted. Something wrong with my commit.